lib/ui/window/pointer_data.h

```diff
@@ -9,6 +9,21 @@
 
 namespace flutter {
 
+// Must match the button constants in events.dart.
+enum PointerButton : int64_t {
+  kPrimaryMouseButton = 1 << 0,
+  kSecondaryMouseButton = 1 << 1,
+  kMiddleMouseButton = 1 << 2,
+  kBackMouseButton = 1 << 3,
+  kForwardMouseButton = 1 << 4,
```

Comment:
Since you're not using enum classes, it's very helpful to have a common prefix for the things that are conceptually grouped. At a minimum I would change these to:
```
kMousePrimaryButton
kMouseSecondaryButton
...
kStylusContact
kStylusPrimaryButton
```
etc.

I'd suggest going further to make it clear they are part of pointer button though, and do:
```
kPointerButtonMousePrimary
kPointerButtonMouseSecondary
...
kPointerButtonStylusContact
kPointerButtonStylusPrimary
```

Reply:
Fixed using the 2nd design